### PR TITLE
moved json to includes for deny test

### PR DESCRIPTION
**Pull request by sparrell**

### CODE REVIEW

File: test/deny_SUITE.erl [hunk 1]
```diff
@@ -134,14 +118,7 @@ test_bad_deny(_Config) ->
...
     %% Deny requires a target and a target - leave off target and it should fail
-    Json = <<"{ \"action\": \"deny\",
-               \"actuator\": {
-                  \"type\": \"network-scanner\",
-                  \"specifiers\": \"scanner01\"
-                           },
-               \"modifiers\": { \"response\": \"ack\",
-                              \"where\": \"perimeter\"}
-             }">>,
+    Json = ?JSON_DENY_WO_TARGET,
```
Comment (gadgetci) [inline test/deny_SUITE.erl]: According to **Elvis**:
> The variable "JSON_DENY_WO_TARGET" on line 121 does not respect the format defined by the regular expression '"^([A-Z][0-9a-zA-Z]*)$"'.

Comment (elvisci) [inline test/deny_SUITE.erl]: The variable "JSON_DENY_WO_TARGET" on line 121 does not respect the format defined by the regular expression '"^([A-Z][0-9a-zA-Z]*)$"'.

File: test/deny_SUITE.erl [hunk 2]
```diff
@@ -68,27 +72,7 @@ test_deny(_Config) ->
...
-\"actuator\": {
-    \"type\": \"network-router\",
-    \"specifiers\": {\"port\": \"2\"}
-    },
-\"modifiers\": {
-    \"response\": \"ack\",
-    \"where\": \"perimeter\"
-    }
-}">>,
+    Json = ?JSON_DENY_01,
```
Comment (gadgetci) [inline test/deny_SUITE.erl]: According to **Elvis**:
> The variable "JSON_DENY_01" on line 75 does not respect the format defined by the regular expression '"^([A-Z][0-9a-zA-Z]*)$"'.

Comment (elvisci) [inline test/deny_SUITE.erl]: The variable "JSON_DENY_01" on line 75 does not respect the format defined by the regular expression '"^([A-Z][0-9a-zA-Z]*)$"'.

**Pull request merged**